src/server/server.ts

```diff
@@ -1,5 +1,7 @@
 // ----- Imports ----- //
 
+import 'source-map-support/register'; // activating the source map suport
```

Comment:
What does the source map do?

Reply:
This intercepts exceptions and "beautify" them with the source map so that in our logs we can get the actual source code line rather than the bundle line.

Reply:
ok 👍

Reply:
```suggestion
import 'source-map-support/register'; // activating the source map support
```